transports/plaintext/Cargo.toml

```diff
@@ -27,3 +27,10 @@ rand = "0.8"
 
 [build-dependencies]
 prost-build = "0.11"
+
+# Passing arguments to the docsrs builder in order to properly document cfg's. 
+# More information: https://docs.rs/about/builds#cross-compiling
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
+rustc-args = ["--cfg", "docsrs"]
```

Comment:
`#![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]` seems to be missing on `/transports/plaintext/lib.rs`

Reply:
added